.github/risk_level_guide.md

```diff
@@ -0,0 +1,23 @@
+## Risk level guide
+
+**Low risk**: with high confidence the change won’t unexpectedly change the behavior, or the change touches a non-critical area.
```

Comment:
How about:

"with high confidence the change won't introduce a bug, or the change touches a non-critical area."?